packages/commons/src/auth/jwk.ts

```diff
@@ -64,3 +66,17 @@ export function sortJWK(jwk: JsonWebKey): JsonWebKey {
       {}
     );
 }
+
+export function getJwksClient(): JwksClient[] {
+  const config = JWTConfig.parse(process.env);
+  return config.wellKnownUrls.map((url) =>
+    jwksClient({
+      cache: true,
+      cacheMaxEntries: 50,
+      timeout: 30000,
+      cacheMaxAge: 3600000, // 60 minutes
+      jwksRequestsPerMinute: 30,
+      jwksUri: url,
+    })
```

Comment:
I would also enable rate limiting setting `rateLimit: true` - citing the docs: "To prevent attackers to send many random kid", see https://github.com/auth0/node-jwks-rsa/blob/master/EXAMPLES.md#rate-limiting